homeassistant/components/incomfort/strings.json

```diff
@@ -31,6 +31,26 @@
       "unknown": "[%key:component::incomfort::config::abort::unknown%]"
     }
   },
+  "options": {
+    "step": {
+      "gateway": {
+        "title": "Intergas InComfort Lan2RF Gateway options",
+        "data": {
+          "legacy_setpoint_status": "Legacy setpoint handling"
+        },
+        "data_description": {
+          "legacy_setpoint_status": "Only enable this option if experience issues in updating the setpoint for your thermostat."
```

Comment:
Since this is the only configuration parameter, I would expect either the description to explain why we have this, who is affected and what this switch will do.

Reply:
I rephrased the data_description